### PR TITLE
SSEE-1581 already enrolled page content update

### PR DESCRIPTION
**Pull request by RachGibbons**

### CODE REVIEW

File: test/uk/gov/hmrc/icedsubscriptionfrontend/views/AlreadyEnrolledPageSpec.scala [hunk 1]
```diff
@@ -57,24 +59,41 @@ class AlreadyEnrolledPageSpec extends SpecBase {
     }
 
     "have a only one page heading" in {
-      document.select(Selectors.h1).text shouldBe "Enrol with the Safety and Security service"
+      document.select(Selectors.h1).text shouldBe "Your organisation is already enrolled with S&S GB"
```
Comment (jeremystone) [inline test/uk/gov/hmrc/icedsubscriptionfrontend/views/AlreadyEnrolledPageSpec.scala:62]: IIRC a heading is not the same as the browser tab title it should form part of the title.

So I think line 49 above should read:
`document.title shouldBe "Your organisation is already enrolled with S&S GB - Enrol with the Safety and Security service - GOV.UK" `

File: test/uk/gov/hmrc/icedsubscriptionfrontend/views/AlreadyEnrolledPageSpec.scala [hunk 2]
```diff
@@ -57,24 +59,41 @@ class AlreadyEnrolledPageSpec extends SpecBase {
...
       document.select(Selectors.h1).size shouldBe 1
     }
 
-    "have inset text for with notice" in {
+    "have indication of when active" in {
       content
         .select(Selectors.paragraph)
         .first
-        .text shouldBe "The GB EORI you supplied is already enrolled with the Safety and Security service."
+        .text shouldBe "The GB EORI you supplied (GB1234567890) is already enrolled with S&S GB."
```
Comment (jeremystone) [inline test/uk/gov/hmrc/icedsubscriptionfrontend/views/AlreadyEnrolledPageSpec.scala:70]: Can we have a test where the eori is not defined (so as to ensure that we don't end up with:

> The GB EORI you supplied () is ...

File: test/uk/gov/hmrc/icedsubscriptionfrontend/services/AuthServiceSpec.scala [hunk 3]
```diff
@@ -115,49 +121,54 @@ class AuthServiceSpec extends SpecBase with MockAuthConnector {
...
-          Enrolments(Set(ssEnrolment.copy(state = "disabled"))) and Some(User) and Some(AffinityGroup.Organisation) and Some(ggwCreds))
+        stubAuth() returns Future.successful(Enrolments(Set(ssEnrolment.copy(state = "disabled"))) and Some(User) and Some(
+          AffinityGroup.Organisation) and Some(ggwCreds))
 
         service.authenticate().futureValue shouldBe AuthResult.LoggedIn(UserType.NotEnrolled)
       }
     }
 
     "user HMRC-SS-ORG enrolment is active" must {
       "return AlreadyEnrolled" in {
```
Comment (jeremystone) [inline test/uk/gov/hmrc/icedsubscriptionfrontend/services/AuthServiceSpec.scala:150]: Maybe add a test where there is an active enrolment but no eori is defined and so get `AlreadyEnrolled(None)`. (Otherwise final case in AuthService `getIdentifier(...)` is not covered.)

File: conf/messages [hunk 4]
```diff
@@ -44,9 +44,15 @@ successfullyEnrolled.findCallcharges=Find out about call charges
...
-alreadyEnrolled.notice=The GB EORI you supplied is already enrolled with the Safety and Security service.
-alreadyEnrolled.makeADeclaration=Make an Entry Summary declaration
+alreadyEnrolled.heading=Your organisation is already enrolled with S&S GB
+alreadyEnrolled.para.1.1=The GB EORI you supplied
+alreadyEnrolled.para.1.2=is already enrolled with S&S GB.
+alreadyEnrolled.para.1.3=You can now use third party software to make an entry summary declaration.
+alreadyEnrolled.csp.heading=If you need help
+alreadyEnrolled.contact=Telephone: 0300 322 7067
+alreadyEnrolled.times=Monday to Friday, 9am to 5pm (except public holidays)
+alreadyEnrolled.findCallcharges=Find out about call charges
```
Comment (jeremystone) [inline conf/messages:54]: Does not appear to be used.
